models/__test__/userModel.test.ts

```diff
@@ -150,5 +150,22 @@ describe("Unit Tests for User Model:", () => {
 
             expect(binChicken).toBe(null);
         });
+
+        // Delete User
+        // --------------------------------------------------------------------
+        test("Should be able to delete own user account", async () => {
+            // First create a user in the database
+            const tim = await users.register({
+                userName: "Tim",
+                email: "tim@gmail.com",
+                password: "TimsStrongPassword",
```

Comment:
validation schema for the password requires a symbol in the password, "Tim-123" may be better. I am not sure if the schema will affect the test result :)

Reply:
It won't have any effect here as this interacts directly with the database - testing the database layer only. It never passes through validation.

Reply:
Thanks for the explanation.